components/transactions/TransactionItem.vue

```diff
@@ -16,15 +16,33 @@
                 v-for="(address, index) in transaction.details.to"
                 :key="address + index"
               >
-                {{ address }}
+                to: {{ address }}
+              </p>
+            </template>
+            <template v-if="transactionCaption === `SendMultiple`">
+              <p>
+                {{ sendMultipleTosCount }}
+              </p>
+            </template>
+            <template v-if="transactionCaption === `ReceiveMultiple`">
+              <p>
+                {{ receiveMultipleSenderAddress }}
               </p>
             </template>
             <template v-if="transactionCaption === `Receive`">
-              <p
-                v-for="(address, index) in transaction.details.from"
-                :key="address + index"
-              >
-                {{ address }}
+              <p>
+                {{ receiveSenderAddress }}
+              </p>
```

Comment:
I think do not change the original logic is better, in case we want to apply update from upstream later